src/lib/viewers/archive/Breadcrumbs.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Breadcrumb from 'box-ui-elements/es/components/breadcrumb';
+import PlainButton from 'box-ui-elements/es/components/plain-button/PlainButton';
+import { VIEWS } from './constants';
+import './Breadcrumbs.scss';
+
+class Breadcrumbs extends React.PureComponent {
+    static propTypes = {
+        fullPath: PropTypes.string.isRequired,
+        onClick: PropTypes.func.isRequired,
+        view: PropTypes.string.isRequired,
+    };
+
+    /**
+     * Split full path string to path items
+     *
+     * @param {string} fullPath - Full path for current folder
+     * @return {Array<Object>} path items including name and path string
+     */
+    getPathItems = fullPath => {
+        const { view } = this.props;
+
+        if (view === VIEWS.VIEW_SEARCH) {
+            return [
+                {
+                    name: __('search_results'),
+                },
+            ];
+        }
+
+        const pathNames = fullPath.split('/').slice(0, -1);
+
+        return pathNames.map((name, index) => ({
+            name,
+            path: `${pathNames.slice(0, index + 1).join('/')}/`,
```

Comment:
The logic for generating these values is a little difficult to follow, which is worrying since it drives the click behavior. Can this be simplified/clarified, maybe with a lodash or path helper?